numpy/core/function_base.py

```diff
@@ -112,7 +112,6 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None,
 
     >>> import matplotlib
     >>> import matplotlib.pyplot
```

Comment:
can you also remove the above two lines in all examples? the only thing that should be needed is `import matplotlib.pyplot as plt`